lib/sass/calcite-web/patterns/_accordion.scss

```diff
@@ -19,25 +19,48 @@
   }
 
   @mixin accordion-title() {
-    @include side-nav-title();
+    @include font-size(0);
+    border: 1px solid $light-gray;
+    margin: -1px 0 0 0;
+    color: $dark-gray;
+    padding: $baseline/2;
     cursor: pointer;
     &:focus,
     &:hover {
       @include transition(all, $transition);
-      background-color: $blk-010;
-      outline: none;
+      border-color: $dark-gray;
+      color: $off-black;
+      position: relative;
```

Comment:
Need it so that the full border displays on hover (border-bottom):
![image](https://user-images.githubusercontent.com/6144462/54960444-db16ec00-4f19-11e9-8039-fd39122cdd24.png)

Reply:
@macandcheese, any insights on how to approach this? if any.